user-service/src/main/java/rs/edu/raf/banka1/model/CurrentAccount.java

```diff
@@ -0,0 +1,25 @@
+package rs.edu.raf.banka1.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class CurrentAccount {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String accountNumber;
+    private Long ownerId;
+    private Double balance;
+    private Double availableBalance;
+    private Long createdByAgentId;
+    private Integer creationDate;
+    private Integer expirationDate;
+    private String currency;
+    private String accountStatus;
+    private String subtypeOfAccount;
```

Comment:
Nitpick: Should subtypeOfAccount be somekind of ENUM since the description of the attribute is as follows:
"Jedan od navedenih: Poslovni, Licni, Stedni, Penzionerski ili Devizni racun za isplatu plata".

Reply:
We will ignore this, as I don't know all types of account that exist.